db/migrate/20230109165658_drop_user_mfa_required_prompt_skip_date.rb

```diff
@@ -0,0 +1,9 @@
+class DropUserMfaRequiredPromptSkipDate < ActiveRecord::Migration[7.0]
+  def up
+    safety_assured { remove_column :users, :non_restricted_mfa_required_prompt_skip_date }
+  end
+
+  def down
+    add_column :users, :non_restricted_mfa_required_prompt_skip_date, :date
+  end
```

Comment:
Non-blocking: We could write this as a single `change` method, which [can support rollback for `remove_column`](https://edgeguides.rubyonrails.org/active_record_migrations.html).

```suggestion
  def change
    safety_assured { remove_column :users, :non_restricted_mfa_required_prompt_skip_date, :date }
  end
```